test/integration/resource_community/inspec/php_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe 'PHP' do
+  it 'has php' do
+    expect(command('php -v').exit_status).to eq(0)
```

Comment:
IMHO you should check what major version of php is installed so you know you got the correct one.